tests/wpt/web-platform-tests/html/semantics/interactive-elements/the-dialog-element/dialog-close.html

```diff
@@ -36,10 +36,8 @@
       was_queued = false;
 
   test(function(){
-    assert_throws("INVALID_STATE_ERR", function() {
-      d1.close();
-    });
-  }, "close() on a <dialog> that doesn't have an open attribute throws an InvalidStateError exception");
+    assert_equals(d1.returnValue, "");
```

Comment:
You should call `d1.close("some value")` before this, otherwise the test just doesn't do anything, right?